app/services/setting_registry.rb

```diff
@@ -0,0 +1,98 @@
+class SettingRegistry
+  include Singleton
+
+  def initialize
+    @settings = {}
+    load
+  end
+
+  def logger
+    Foreman::Logging.logger('app')
+  end
+
+  def find(name)
+    @settings[name.to_s]
+  end
+
+  # Returns a setting effective value
+  # Call as `Foreman.settings[<name>]`
+  #
+  #   Foreman.settings['default_locale'] => nil.
+  #   Foreman.settings[:default_locale] => nil.
+  #
+  def [](name)
+    find(name)&.value
```

Comment:
Should we raise an exception if there is no such setting? Or at least log it?

Reply:
We can log it, exception would make sense, but there are :dragon:s and it would change from previous implementation.